Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -232,6 +232,20 @@
 )
 reconstruction_ecalOnly = cms.Sequence(reconstruction_ecalOnlyTask)
 
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitHBHE_cfi import particleFlowRecHitHBHE as _particleFlowRecHitHBHE
+_particleFlowRecHitHBHE.producers[0].src=cms.InputTag("hbheprereco","")
```

Comment:
this looks like a wrong place to modify ```particleFlowRecHitHBHE```

Reply:
yes, rather than importing and renaming, probably better to make a clone for the hcalOnly task

Reply:
IMHO, it would be better to manipulate specific modules in the subsystem-specific place, not to pollute the central Configuration/StandardSequences with lower level details.

Reply:
Core advocates an object to be modified only in the file where it is defined. Otherwise the object state depends on whether another configuration fragment is loaded in the Process or not (I know we do that currently).

Reply:
indeed, the clone can be created in `particleFlowRecHitHBHE_cfi.py` and then just imported here

Reply:
sure will move this cloning, and likely need to be eras dependent again.
Will do after the https://github.com/cms-sw/cmssw/pull/30138 is merged

Reply:
see last commit for this change #083efc5